Client/my-story-illustrator-client/src/index.js

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
```

Comment:
It is customary in a React app to keep the index.js file super super minimal. It is only used to render stuff in App.js. Look at what we have in the main branch for index.js. Let's make this look the same as what is in the main branch right now. Then any code you want to display on the page, put it in the App.js file instead.

One more thing: After you have moved everything into App.js, put the <ChakraProvider> wrapper around everything that is returned from app.js. If we don't have that element wrapped around the root component, some of the Chakra styles will not display correctly.